docker-compose.yml

```diff
@@ -39,6 +39,7 @@ services:
      OO_HELP_EMAIL: "info@lucyparsonslabs.com"
      OO_SERVICE_EMAIL: "openoversightchi@lucyparsonslabs.com"
      S3_BUCKET_NAME: "${S3_BUCKET_NAME}"
+     TIMEZONE: "America/Chicago"
```

Comment:
This has a default value in the config file, but I put it here as a signal to remind anyone deploying their own app to make sure to update the value. It's more of a signal than something that's absolutely needed for the Chicago instance.